packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 46.11.0 - August 22, 2019
+
+## Fixed
+
+#### salesforcedx-core
+
+- Add custom objects as a bundle for retrieving in the Org Browser ([PR #1569](https://github.com/forcedotcom/salesforcedx-vscode/pull/1569))
+
+#### docs
+
+- Fix Org Browser documentation link ([PR #1565](https://github.com/forcedotcom/salesforcedx-vscode/pull/1565))
```

Comment:
Hi @sfsholden , do you want to specify where the doc link was broken? 
Fix Org Browser documentation link in <...........>

Reply:
Sounds good, give me a moment and I'll update it.